src/wav2vec_toolkit/utils.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+import textwrap
+
+import pkg_resources
+
+
+BASE_PATH = "wav2vec_toolkit"
+LANG_PATH = "languages"
+LANG_MODULE_REQUIREMENTS = ["normalizer.py", "parameters.py", "requirements.txt"]
+
+
+def get_file_path(name: str):
+    return os.path.abspath(os.path.join(os.path.dirname(__file__), name))
+
+
+def parse_requirements(filename: str):
+    lineiter = (line.strip() for line in open(filename))
+    return [line for line in lineiter if line and not line.startswith("#")]
+
+
+def load_module_from_lang(lang: str):
+    lang_mod_path = f"{BASE_PATH}/{LANG_PATH}/{lang}"
```

Comment:
Have you checked that this works when the package is installed via `pip`?

Reply:
Not actually, but I've checked now and found the bug and fixed it.

Reply:
I've just added the parameters for storing the hyperparams, but since @borisdayma uses the hyperparam_search.py, there's no need to add this part.

Reply:
@anton-l, I'll update some minor bugs in the next commit, improve some info, and add the instruction like what Huggingface-datasets did.